core/test_blockchain.go

```diff
@@ -880,9 +900,8 @@ func TestReorgReInsert(t *testing.T, create func(db ethdb.Database, gspec *Genes
 	// Ensure that key1 has some funds in the genesis block.
 	genesisBalance := big.NewInt(1000000000)
 	gspec := &Genesis{
-		Config:   &params.ChainConfig{HomesteadBlock: new(big.Int), FeeConfig: params.DefaultFeeConfig},
-		Alloc:    GenesisAlloc{addr1: {Balance: genesisBalance}},
-		GasLimit: params.DefaultFeeConfig.GasLimit.Uint64(),
+		Config: &params.ChainConfig{HomesteadBlock: new(big.Int)},
+		Alloc:  GenesisAlloc{addr1: {Balance: genesisBalance}},
```

Comment:
This could panic without a FeeConfig and GasLimit (if we don't check gas limit in subnet-evm only)